emission/tests/analysisTests/intakeTests/TestUserStat.py

```diff
@@ -75,7 +75,7 @@ def testGetAndStoreUserStats(self):
         self.assertIn("pipeline_range", profile, "User profile should contain 'pipeline_range'.")
         self.assertIn("last_call_ts", profile, "User profile should contain 'last_call_ts'.")
 
-        expected_total_trips = 5
```

Comment:
the commit that added this (https://github.com/e-mission/e-mission-server/pull/1006/commits/8a9939cf7d4274e016668b08b6bf818442d7a598) should have had a short explanation justifying the change.

I _thought_ this was because you are running the pipeline a second time (`etc.runIntakePipeline(self.testUUID)`). But you are not loading any new data in these changes *and* we check the variable before the new pipeline run
https://github.com/e-mission/e-mission-server/pull/1006/files#diff-69c9cbc3241bd93c4c5cae6866fec4189a9aa32c8964e86b2bdc3749bae4d2b2L82

So why does this need to change? Please investigate and provide justification.